projects/plugins/jetpack/modules/search/class-jetpack-search-customize.php

```diff
@@ -153,12 +153,12 @@ public function customize_register( $wp_customize ) {
 			$id,
 			array(
 				'label'       => __( 'Result Format', 'jetpack' ),
-				'description' => __( 'Choose how the search results look.', 'jetpack' ),
+				'description' => __( 'Choose how the search results look. The expanded view will feature images associated with the search result.', 'jetpack' ),
```

Comment:
I think we might be using a sledgehammer to crack a walnut here :) I think the "(shows images)" in the dropdown menu is sufficient and don't think we need to change the description necessarily.

Reply:
I think you may be right! I'm more than fine with just having "displays images" within the dropdown!

Reply:
Reverted!